profiles/base/use.mask

```diff
@@ -105,10 +104,6 @@ prefix
 prefix-guest
 prefix-stack
 
-# Diego E. Pettenò <flameeyes@gentoo.org> (2010-10-19)
-# masking flags that only apply to FreeBSD
-kqueue
```

Comment:
Seems harmless but just to note removing the mask enables FreeBSD stuff rather than disable.

iroffer-dinoex[kqueue] will automagic no-op:
`Checking for kqueue() ... missing, won't use kqueue()`
And squid[kqueue] ebuild still checks for kernel_FreeBSD making it a no-op as well.

Flags existing feels kind of pointless though.

Anyhow, PR looks good to me.